composer.lock

```diff
@@ -51,20 +50,20 @@
                 "event-dispatcher",
                 "event-emitter"
             ],
-            "time": "2012-11-02 14:49:47"
+            "time": "2012-11-02T14:49:47+00:00"
         },
         {
             "name": "monolog/monolog",
-            "version": "1.17.2",
+            "version": "1.22.0",
```

Comment:
Changes to `composer.lock` should only reflect what was changed in `composer.json` (i.e. that `react/socket-client` was upgraded).